static/examples/cstg-secure-signals-example/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavaScript SDK, Secure Signals</title>
```

Comment:
Is it fine that we're saying UID2 here but we're on the EUID site? Didn't notice that before sorry. Lines 5, 15, 21. Just checking.

Reply:
changed to EUID